.github/workflows/rust-tests.yml

```diff
@@ -56,18 +50,38 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: install opensc and dependencies
-        run: apt-get update && apt-get install -y opensc curl libssl-dev build-essential pkg-config xxd jq
+        run: apt-get update && apt-get install -y opensc curl libssl-dev build-essential pkg-config xxd jq gnutls-bin
```

Comment:
"apt" instead of "apt-get" is the recommended tool. :-)

Reply:
When running `apt` with no terminal/in scripts you get this warning : 

```
WARNING: apt does not have a stable CLI interface. Use with caution in scripts.
```

I prefer staying on the "stable" apt-get interface